drishyauk/assessment-string-calculator
# feat:created string calculator assessment

### Steps to test:
- Clone the repo
- cd <repo-folder-name>
- npm i
- npm run dev
- Open the localhost in browser

<img width="1222" alt="Screenshot 2024-11-26 at 2 40 54 PM" src="https://github.com/user-attachments/assets/3e9f6c7e-68a6-45f5-acb4-902fa2369061">


- enter nothing, click Calculate button, Observe that the result shows 0
- Enter 2 numbers separated by comma, click on Calculate button, Observe that appropriate result is displayed
- Enter a number, now press enter key, enter another number(s), click on Calculate button, Observe the result is displayed
- Enter //<delimiter><press enter><enter numbers seperated by comma>, click on Calculate, observe the result. (for ex: //;::::\n1,2)
- Enter negative numbers, click Calculate, Observe that appropriate error msg is displayed with the negative numbers.

<img width="775" alt="Screenshot 2024-11-26 at 2 45 46 PM" src="https://github.com/user-attachments/assets/66ba3686-8259-4ffd-975c-3dd49c6d5478">
